src/irmin-pack/unix/gc.ml

```diff
@@ -282,7 +282,9 @@ module Make (Args : Args) : S with module Args := Args = struct
       in
       let* () =
         Errors.finalise (fun _outcome ->
-            Ao.close prefix |> Errs.log_if_error "GC: Close prefix")
+            Ao.fsync prefix
+            >>= (fun _ -> Ao.close prefix)
+            |> Errs.log_if_error "GC: Close prefix")
```

Comment:
You have 2 fsyncs for `prefix`.

Reply:
Ah, good catch. Only need to fsync the last interaction.